types/model.d.ts

```diff
@@ -115,15 +282,71 @@ export declare class ModelDefinition extends EventEmitter implements Schema {
   toJSON(forceRebuild?: boolean): AnyObject;
 }
 
+export interface ModelBaseClassOptions {
+  applySetters?: boolean;
+  applyDefaultValues?: boolean;
+  strict?: boolean;
+  persisted?: boolean;
+}
+
+interface ModelMergePolicy {
+  description?: {
+    replace: boolean;
+  };
+  properties?: {
+    patch: boolean;
+  }; // Only referenced in "legacy built-in merge policy"
+  options?: {
+    path: boolean;
```

Comment:
This looks like a typo
```suggestion
    patch: boolean;
```